Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+
+**Desirable features**
+
+- Additionally supported artifacts
+  - Additional artifacts that the registry is able to store in addition to OCI artifacts, e.g. Java, Node.js, or Python packages
+- Integration possibilities
+  - Ability to cooperate with another software solution in order to improve own feature set (e.g. integration of P2P solution for improving container image distribution (download speed and stability, high scalability ...))
+
+Refer to the table of evaluated projects with their features. Note that only container
+registry implementations that passed the OSS health stage (Harbor, Quay, and Dragonfly)
+are evaluated here.
+
+| Features                            | Harbor                                        | Quay                                                                | Dragonfly                    |
+|-------------------------------------|-----------------------------------------------|---------------------------------------------------------------------|------------------------------|
+| Audit Logs                          | ✓                                             | ✓                                                                   | ✗                            |
+| Authentication of system identities | ✓ Robot Accounts                              | ✓ Robot Accounts                                                    | ✗                            |
+| Authentication of users             | ✓ Local database, LDAP, OIDC, UAA             | ✓ Local database, LDAP, Keystone, JWT                               | ✓ Local database             |
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
+| Vulnerability scanning              | ✓ Trivy, Clair                                | ✓ Clair                                                             | ✗                            |
+| Content Trust and Validation        | ✓ Cosign                                      | ✓ Cosign                                                            | ✗                            |
+| Multi-tenancy                       | ✓ (not on the storage level)                  | ✓ (not on the storage level)                                        | ✓ (not on the storage level) |
+| Backup and restore                  | ✓                                             | ✓                                                                   | ✗                            |
+| Monitoring                          | ✓ Prometheus metrics                          | ✓ Prometheus metrics                                                | ✓ Prometheus metrics         |
```

Comment:
Harbor also supports tracing

Reply:
I incorporated it in 02382f14b5ba7204845eb3e690c4d5b78a49770b